README.md

```diff
@@ -64,8 +64,23 @@ ncalls tottime  percall  cumtime  percall filename:lineno(function)
 2    0.000    0.000    0.306    0.153 C:\Users\ravish.ranjan\AppData\Local\Programs\Python\Python36\lib\site-packages\urllib3\connectionpool.py:446(urlopen)
 2    0.000    0.000    0.306    0.153 C:\Users\ravish.ranjan\AppData\Local\Programs\Python\Python36\lib\site-packages\urllib3\connectionpool.py:319(_make_request)
 1    0.000    0.000    0.258    0.258 C:\Work\PycharmProjects\node-notification-handler\service\portal.py:34(get_subscription_members)
-..............................
-................................
+```
+
+**Sesam Logger**
+
+ You can use this to save time and lines of code. It provides the standard customization of root level logging- 
+ configuration and gives you SESAM uniform standard settings.
+
+```python
+from sesamlogger import sesam_logger
```

Comment:
the example should import from sesamutils

Reply:
fixed.